types/node/worker_threads.d.ts

```diff
@@ -98,7 +124,7 @@ declare module "worker_threads" {
          */
         constructor(filename: string | URL, options?: WorkerOptions);
```

Comment:
According to the docs, [`WorkerOptions.transferList`](https://nodejs.org/docs/latest/api/worker_threads.html#worker_threads_new_worker_filename_options) also supports `FileHandle` now.

Reply:
Done!